examples/apps/color_finder.py

```diff
@@ -0,0 +1,710 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2017 Anki, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License in the file LICENSE.txt or at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS, 
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''Cozmo looks around and drives after colors.
+
+When the program starts, Cozmo will look around for the color blue.
+Tap the cube illuminated blue to switch Cozmo's target color between blue, red, green, and yellow.
+Tap the blinking white cube to have the viewer display Cozmo's pixelated camera view.
+
+'''
+
+import asyncio
+import functools
+import math
+import numpy
+import sys
+
+import cozmo
+
+from cozmo.util import degrees, distance_mm, radians, speed_mmps
+from cozmo.lights import Color, Light
+try:
+    from PIL import Image, ImageColor, ImageDraw
+except ImportError:
+    sys.exit('Cannot import from PIL: Do `pip3 install --user Pillow` to install')
+    
+# map_color_to_light (dict): maps each color name with its cozmo.lights.Light value.
+# Red, green, and blue lights are already defined as constants in lights.py, 
+# but we need to define our own custom Light for yellow.
+
+map_color_to_light = {
+'green' : cozmo.lights.green_light, 
+'yellow' : Light(Color(name='yellow', rgb = (255, 255, 0))), 
```

Comment:
I wonder if it might be worth making yellow, cyan, and magenta defined colors in the lights class itself - I don't think we want to dive too deep down the "define all the things" path, but primary+secondary colors seems like a reasonable set.